atd-etl/cr3_download/process/helpers_cr3.py

```diff
@@ -56,26 +46,29 @@ def download_cr3(crash_id, cookies):
     for key, morsel in cookie.items():
         baked_cookies[key] = morsel.value
 
-    crash_id_encoded = base64.b64encode(str("CrashId=" + crash_id).encode("utf-8")).decode("utf-8")
+    crash_id_encoded = base64.b64encode(
+        str("CrashId=" + crash_id).encode("utf-8")
+    ).decode("utf-8")
     url = ATD_ETL_CONFIG["ATD_CRIS_CR3_URL"] + crash_id_encoded
-    download_path = ATD_ETL_CONFIG["AWS_CRIS_CR3_DOWNLOAD_PATH"] + "%s.pdf" % crash_id
```

Comment:
This path was coming from the env sometimes and hardcoded in other places. I decided to hardcode it to `/tmp/` since it is unlikely to need change from the outside of the script.